client/layout/logged-out.jsx

```diff
@@ -134,7 +133,7 @@ const LayoutLoggedOut = ( {
 					{ secondary }
 				</div>
 			</div>
-			{ config.isEnabled( 'gdpr-banner' ) && <GdprBanner showGdprBanner={ showGdprBanner } /> }
+			{ config.isEnabled( 'gdpr-banner' ) && <SideloadAnalyticsBanners /> }
```

Comment:
I just tested `/plugins` with JS disabled to see a SSR render and it doesn't appear. This might cause a LCP metric regression - can we try supporting it on SSR as well?